AccountingService/Controllers/UsersController.cs

```diff
@@ -23,7 +26,7 @@ public UsersController(SalaryConversionContext context) : base()
         }
 
         [HttpGet]
-        
+        [Authorize]
```

Comment:
This should be applied to the whole controller. So all endpoints are only accessed if the user is authorized.